opentech/apply/funds/templates/funds/includes/review_sidebar.html

```diff
@@ -0,0 +1,47 @@
+{% load review_tags %}
+<ul class="reviews-sidebar js-reviews-sidebar">
+    {% if reviews_exist %}
+        <li class="reviews-sidebar__item reviews-sidebar__item--header">
+            <div></div>
+            <div>{{ recommendation|traffic_light }}</div>
+            <div></div>
+        </li>
+    {% endif %}
+
+    {% if not assigned_staff %}
+        <li class="reviews-sidebar__no-reviews">No reviews yet</li>
+    {% endif %}
+
+    {% for review_data in reviews_block.role_reviewed %}
+        {% include 'funds/includes/review_sidebar_item.html' with review=review_data.review reviewer=review_data.reviewer role=review_data.role opinions=review_data.opinions %}
+    {% endfor %}
+
+    {% for review_data in reviews_block.staff_reviewed %}
+        {% include 'funds/includes/review_sidebar_item.html' with review=review_data.review reviewer=review_data.reviewer opinions=review_data.opinions %}
+    {% endfor %}
+
+    {% for review_data in reviews_block.role_not_reviewed %}
+        {% include 'funds/includes/review_sidebar_item.html' with reviewer=review_data.reviewer role=review_data.role missing=True %}
+    {% endfor %}
+
+    {% for review_data in reviews_block.staff_not_reviewed %}
+        {% include 'funds/includes/review_sidebar_item.html' with reviewer=review_data.reviewer missing=True %}
+    {% endfor %}
+
+    {% if object.stage.has_external_review %}
+
+    {% if reviews_block.external_reviewed or reviews_block.external_not_reviewed %}
+        <hr class="reviews-sidebar__split">
+            {% for review_data in reviews_block.external_reviewed %}
+                {% include 'funds/includes/review_sidebar_item.html' with review=review_data.review reviewer=review_data.reviewer opinions=review_data.opinions %}
+            {% endfor %}
+
+            {% for review_data in reviews_block.external_not_reviewed %}
+                {% include 'funds/includes/review_sidebar_item.html' with reviewer=review_data.reviewer missing=True class="hidden"  %}
+            {% endfor %}
+
+            <li><a class="link link--bold link--underlined js-toggle-reviewers" href="#">All Assigned Advisors</a></li>
+        {% endif %}
+    {% endif %}
```

Comment:
This was probably my fault in the original file that this was copied from, but these two `{% endif %}` tags are indented incorrectly

Reply:
good spot! will update